features/matchers/have_enqueued_job_matcher.feature

```diff
@@ -1,6 +1,6 @@
 Feature: have_enqueued_job matcher
 
-  The `have_enqueued_job` matcher is used to check if given ActiveJob job was enqueued.
+  The `have_enqueued_job` (also known as `enqueue_job`) matcher is used to check if given ActiveJob job was enqueued.
```

Comment:
Just to be as clear as possible here, I would suggest saying "(also aliased as `enqueue_job`)"

"known as" could just mean that people refer to it in conversation by that name.